hail_scripts/v02/utils/elasticsearch_client.py

```diff
@@ -29,14 +29,15 @@ def export_table_to_elasticsearch(
         delete_index_before_exporting :bool = True,
         elasticsearch_write_operation :str = ELASTICSEARCH_INDEX,
         ignore_elasticsearch_write_errors :bool = False,
-        elasticsearch_mapping_id=None,
+        elasticsearch_mapping_id :str = "docId",
```

Comment:
This should be set in Seqr loading code instead of changing the default. This is also used for loading gnomAD datasets, none of which have a `docId` field.

Reply:
Beat me to it haha.